Source/Parallelization/GuardCellManager.H

```diff
@@ -37,6 +38,13 @@ public:
      * \param nci_corr_stencil stencil of NCI corrector
      * \param maxwell_solver_id if of Maxwell solver
      * \param max_level max level of the simulation
+     * \param v_galilean ...?
+     * \param v_comoving ...?
+     * \param safe_guard_cells ...?
+     * \param do_electrostatic ...?
+     * \param do_multi_J ...?
+     * \param fft_do_time_averaging ...?
```

Comment:
@RemiLehe or @EZoni maybe? :)
```suggestion
     * \param v_galilean ...?
     * \param v_comoving ...?
     * \param safe_guard_cells ...?
     * \param do_electrostatic ...?
     * \param do_multi_J ...?
     * \param fft_do_time_averaging ...?
```